packages/block-editor/src/components/link-control/style.scss

```diff
@@ -147,7 +153,6 @@ $block-editor-link-control-number-of-actions: 1;
 		text-overflow: ellipsis;
 		max-width: 230px;
 		overflow: hidden;
-		white-space: nowrap;
```

Comment:
The three styles `text-overflow`, `overflow`, and `white-space` are pretty directly linked, if I understand it correctly. I would expect that either all should be removed, or all should remain.

What was the purpose for removing `white-space`? As best I can tell, in combination with the other styles, it would not been the case that the element would stretch, and instead it would be truncated.

![image](https://user-images.githubusercontent.com/1779930/74373366-4d9aef80-4daa-11ea-9520-05055506db46.png)

Contrast that with how it behaves after these changes:

![image](https://user-images.githubusercontent.com/1779930/74373416-6a372780-4daa-11ea-9c35-d1f6d8ce96c1.png)

I think either _could_ work, but it seems like the behavior of the former was likely a conscious design decision to keep the heights of these suggestions consistent (cc @getdave).

I'd guess this might have been related to my comment at https://github.com/WordPress/gutenberg/pull/20154#issuecomment-584347182 ? It feels to me we shouldn't want there to be an explicitly assigned `max-width` here, at least not one with a fixed pixel value. I'd guess this also hurts reusability of the component, since it seems pretty focused on it being rendered in the popover.

Reply:
I removed it because the header still stretches up to 230px with it.

Reply:
Yes it was deliberate to ensure consistent height 

Sorry i dont get these pings. I need better GH skills!

Reply:
It's not working for me in Chrome. The content is cut off without an ellipsis.